charger/eebus.go

```diff
@@ -42,6 +43,7 @@ type EEBus struct {
 	minMaxG func() (minMax, error)
 
 	communicationStandard model.DeviceConfigurationKeyValueStringType
+	vasVW                 bool // wether the EVSE supports VW VAS with ISO15118-2
```

Comment:
Was _ist_ VW VAS und warum hängt es vom Charger (statt vom Fahrzeug) ab?

Reply:
Das sind Herstellerspezifische Erweiterungen des ISO15118-2 Protokolls. Das nutzt der Taycan mit der PMCC "Wallbox". Die Erweiterungen sind nicht öffentlich und damit nur mit bestimmten Kombinationen nutzbar.

Reply:
> Diese spezielle Erweiterung ermöglicht die Übertragung des SoC und das Laden ohne dass das Auto selbst einen Ladebedarf hat weil das im Auto eingestellte Ladeziel schon erreicht wurde.

Ladeziel und -limit sind dann aber unabhängig von einander? Wir würden ja nicht ein ggf. im Fahrzeug gesetztes 80% Limit übersteuern wollen?

Reply:
Die Fahrzeuge haben ein Min-SoC der gesetzt wird. Ein Max-SoC gibt es nur über Timer. Der Min-SoC ist per default auf 25%. Ein Ladelimit gibt es hier nicht wenn es eine ISO VAS Kommunikation gibt. Es gibt hier also nur Ladeziele, keine Limits.

Wenn man einen PMCC nutzt, ermöglicht das evcc in evcc das Ladeziel zu definieren, PV zu Laden ohne dass im Fahrzeug ein Timer gesetzt werden muss. Und gleichzeitig dass das Auto auch einschläft! Die Werte werden auch anders an das Auto übertragen als die Obligations.

Wenn man nur die "üblichen" Obligations nutzt und das Laden sperrt, dann schläft das Auto nicht ein. Denn es gibt bei ISO keine Definition/Mechanismus vom Protokoll es schlafen zu schicken.

Daher gibt es diese Unterscheidung für diesen speziellen Lademechanismus. Ich nutze den seit Jahren genau so und möchte dass das weiter so möglich ist. Der Vorteil von diesem geänderten Ansatz ist, dass das Auto auch wirklich einschlafen kann.

Reply:
> Wenn man nur die "üblichen" Obligations nutzt und das Laden sperrt, dann schläft das Auto nicht ein. Denn es gibt bei ISO keine Definition/Mechanismus vom Protokoll es schlafen zu schicken.

Die Obligations bedeuten ja erstmal nicht, dass das Fahrzeug aktiv etwas tun muss/sollte. Ist das nicht ein Fehler der Fahrzeug-Firmware?

> Ich nutze den seit Jahren genau so und möchte dass das weiter so möglich ist. 

Spricht nichts gegen. Da ich es nirgendwo finden konnte: was heisst das Akronym `VAS`? Gibts dazu irgendeine Referenz? Vielen Dank!

Reply:
Nein, es gibt hier keinen Fehler in der Firmware. Die Entwickler haben hier einfach weiter gedacht, wie man PV Laden direkt im Protokoll unterstützen kann und das umgesetzt. 

VAS = Vendor Added Services. Ist Teil der ISO 15118-2 Spezifikation.